### PR TITLE
chore(tier4_perception_launch): enable to receive argument `centerpoint_model_name` from autoware_launch

### PR DESCRIPTION
**Pull request by MasatoSaeki**
## Description
This PR enable to receive argument `centerpoint_model_name` from `autoware_launch`.

This PR need to merge following autoware_launch PR at same time.
- https://github.com/autowarefoundation/autoware_launch/pull/1182

## Related links

**Parent Issue:**

- Link

## How was this PR tested?
I have checked it work correctory by sample rosbag in my local pc.
And also checked whether it can change model. (A part of model is failed but it is no problem because it depend on another repos.)
- [Internal link](https://evaluation.ci.tier4.jp/evaluation/reports/9512d9a2-8d98-5b5a-9b78-12175c31a31e?project_id=prd_jt)

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: launch/tier4_perception_launch/launch/perception.launch.xml [hunk 1]
```diff
@@ -37,9 +37,13 @@
   <arg name="traffic_light_arbiter_param_path"/>
 
   <!-- CenterPoint model parameters -->
-  <arg name="centerpoint_model_name" default="centerpoint_tiny" description="options: `centerpoint`, `centerpoint_tiny` or `centerpoint_sigma`"/>
+  <arg name="centerpoint_model_name" default="$(var centerpoint_model_name)"/>
```
Comment (technolojin) [inline launch/tier4_perception_launch/launch/perception.launch.xml:40]:
```suggestion
  <arg name="centerpoint_model_name"/>
```

File: launch/tier4_perception_launch/launch/perception.launch.xml [hunk 2]
```diff
@@ -37,9 +37,13 @@
...
   <arg name="centerpoint_model_path" default="$(var data_path)/lidar_centerpoint"/>
 
+  <!-- CenterPoint model parameters -->
+  <arg name="transfusion_model_name" default="$(var transfusion_model_name)"/>
```
Comment (technolojin) [inline launch/tier4_perception_launch/launch/perception.launch.xml:44]:
```suggestion
  <arg name="transfusion_model_name"/>
```

**Review by YoshiRi — APPROVED** [review]
In my humble opinion, the default model names in [lidar_dnn_detector.launch.xml](https://github.com/autowarefoundation/autoware.universe/pull/9003/files#diff-174cd66ea65cd31445e77661abe9e17d4882333ed8133ee5d73a08029be0e8a2) seems to be slightly difficult to find out, but it will works.

LGTM for now

**Review by technolojin — APPROVED** [review]
LGTM

**Pull request merged**